Gemfile

```diff
@@ -37,6 +37,8 @@ gem 'rails_12factor', group: :production
 gem 'globalize'
 gem 'globalize-accessors'
 
+gem 'friendly_id'
```

Comment:
I personally love friendly_id because of the way we can define how to generate slugs and manage uniqueness. If you need some help to try both solutions with Globalize, I can help. Ping me on slack and I'll answer when I am available.